plugins/wasi_nn/CMakeLists.txt

```diff
@@ -1,6 +1,11 @@
 # SPDX-License-Identifier: Apache-2.0
 # SPDX-FileCopyrightText: 2019-2022 Second State INC
 
+# add backends building flags
+message(STATUS "Build OpenVINO backend for WASI-NN")
+add_definitions(-DWASMEDGE_WASINN_BUILD_OPENVINO)
```

Comment:
Use `WASMEDGE_WASINN_BACKEND="OpenVINO;..."` is better.

Reply:
Done.